autocomplete.js

```diff
@@ -43,16 +43,26 @@ function getDate(paramName){
   return async (query, pluginSettings, pluginActionParams) => {
     const params = mapAutoParams(pluginActionParams);
     query = query || params[paramName];
-    let queryTime = new Date().toISOString().split("T")[0];
+   // let queryTime = new Date().toISOString().split("T")[0];
+    let date = new Date();
+    queryTime= (((date.getMonth() > 8) ? (date.getMonth() + 1) : ('0' + (date.getMonth() + 1))) + '/' + ((date.getDate() > 9) ? date.getDate() : ('0' + date.getDate())) + '/' + date.getFullYear());
+  
     if (query){
       try {
-        queryTime = new Date(query).toISOString().split("T")[0];
+       // queryTime = new Date(query).toISOString().split("T")[0];
+        date = new Date(query);
+        queryTime= (((date.getMonth() > 8) ? (date.getMonth() + 1) : ('0' + (date.getMonth() + 1))) + '/' + ((date.getDate() > 9) ? date.getDate() : ('0' + date.getDate())) + '/' + date.getFullYear());
```

Comment:
Same here as in the suggested change above, however this will not be necessary once the query time is computed once only.